sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1463,7 +1463,14 @@ def init(self):
         # Initialize xcvr table helper
         xcvr_table_helper = XcvrTableHelper()
 
-        self.load_media_settings()
+        fastboot_tbl = swsscommon.Table(xcvr_table_helper.state_db_host, 'FAST_REBOOT')
+        keys = fastboot_tbl.getKeys()
+
+        if "system" in keys:
+            self.log_info("Skip loading media_settings.json in case of fast-reboot")
+        else:
+            self.load_media_settings()
```

Comment:
the changes made as part of https://github.com/Azure/sonic-utilities/pull/1910 only makes a backup of media settings  file and not transceiver info.  Skipping notification altogether will require us to backup transceiver info as well. Since transceiver info notification is not impacting fast-reboot or causing a flap, is there a need for this change?